dask/highlevelgraph.py

```diff
@@ -211,6 +147,77 @@ def get_dependencies(self, key: Hashable, all_hlg_keys: Iterable) -> set:
         """
         return keys_in_tasks(all_hlg_keys, [self[key]])
 
+    def __dask_distributed_anno_pack__(self) -> Optional[Mapping[str, Any]]:
```

Comment:
I personally find it much more clear when we use the full `annotations` name instead of using `anno` as a shorthand. My thinking is the name of this method is long either way, so we might as well make it totally clear too

```suggestion
    def __dask_distributed_annotations_pack__(self) -> Optional[Mapping[str, Any]]:
```